images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -219,6 +219,12 @@ function Get-GHCVersion {
     return "GHC $ghcVersion"
 }
 
+function Get-GHCupVersion {
+    $(ghcup --version) -match "version v(?<version>\d+\.\d+\.\d+)" | Out-Null
```

Comment:
nitpick: if version can be like 8.3.4.5 than it's better to use `\d+(\.\d+){2,}`

Reply:
GHC's versions are always `a.b.c`, however Chocolatey can (and does) use `a.b.c.d` for patch updates to chocolatey's installation itself as all versions are immutable in Chocolatey.

(It doesn't change the correctness of this in any way)